### PR TITLE
EPMWL-4.3 Add username and login fields to UserOrderDAO methods

### PR DESCRIPTION
**Pull request by MrToady**

### CODE REVIEW

**Review by AleksandrRadchenko — COMMENTED** [review]
I think we don't need to change DAO for this. If we do, we must do this for every complex request. IMHO complex requests should be processed in servlets using several DAO methods.

**Comment by AleksandrRadchenko** [comment]
Reviewed.

File: src/main/java/com/epam/wl/dao/UserOrderDAO.java [hunk 1]
```diff
@@ -20,9 +20,15 @@
 
     public static final String UPDATE_NEW = "INSERT INTO user_order (userid, bookid, status) VALUES(?, ?,'NEW');";
     public static final String UPDATE_STATUS = "UPDATE user_order SET status = ? WHERE id = ?";
-    public static final String QUERY_ALL = "SELECT * FROM user_order";
-    public static final String QUERY_BY_ID = "SELECT * FROM user_order WHERE id = ?";
-    public static final String QUERY_BY_STATUS = "SELECT * FROM user_order WHERE status = ?";
+    public static final String QUERY_ALL = "select user_order.id, user.id, user.name, user.lastname, user.email, " +
```
Comment (ginsenghillock) [inline src/main/java/com/epam/wl/dao/UserOrderDAO.java]: SQL keywords should be in uppercase

File: src/test/java/com/epam/wl/dao/BookOrderDAOTest.java [hunk 2]
```diff
@@ -41,7 +41,9 @@ void setUp() throws SQLException {
     @Test
     void createOneRow() throws SQLException {
         BookInstance bookInstance = new BookInstance(1, 1);
-        UserOrder userOrder = new UserOrder(1,1, 1, UserOrderStatus.IN_PROGRESS);
+        UserOrder userOrder = new UserOrder(
+                1, 1, "Иван", "Иванов", "ivan@ivan.ru", "Азбука",
+                "Петр Иванов", 1954, UserOrderStatus.IN_PROGRESS);
         BookOption bookOption = BookOption.SUBSCRIPTION;
         bookOrderDAO.create(bookInstance, userOrder, bookOption);
```
Comment (ginsenghillock) [inline src/test/java/com/epam/wl/dao/BookOrderDAOTest.java]: Why you pass whole entities to DAO method? You need only ids and option

Comment (AleksandrRadchenko) [reply src/test/java/com/epam/wl/dao/BookOrderDAOTest.java]: Fixed

File: src/main/java/com/epam/wl/entities/UserOrder.java [hunk 3]
```diff
@@ -7,7 +7,12 @@
 @Data @AllArgsConstructor
```
Comment (ginsenghillock) [inline src/main/java/com/epam/wl/entities/UserOrder.java]: It would be better to start each annotation from new line

Comment (AleksandrRadchenko) [reply src/main/java/com/epam/wl/entities/UserOrder.java]: Fixed

**Comment by ginsenghillock** [comment]
Reviewed

**Pull request merged**